exe/kubernetes-deploy

```diff
@@ -43,16 +45,23 @@ revision = ENV.fetch('REVISION') do
   exit 1
 end
 
-KubernetesDeploy::Runner.with_friendly_errors do
+namespace = ARGV[0]
+context = ARGV[1]
+logger = KubernetesDeploy::Logger.build(namespace, context, verbose_tags: verbose_log_tags)
+
   runner = KubernetesDeploy::Runner.new(
-    namespace: ARGV[0],
-    context: ARGV[1],
+    namespace: namespace,
+    context: context,
     current_sha: revision,
     template_dir: template_dir,
-    wait_for_completion: !skip_wait,
+    bindings: bindings,
+    logger: logger
+  )
+
+KubernetesDeploy::Runner.with_friendly_errors do
+  runner.run(
+    skip_wait: skip_wait,
     allow_protected_ns: allow_protected_ns,
-    prune: prune,
-    bindings: bindings
+    prune: prune
```

Comment:
I split these out simply because the number of args to the initializer was getting crazy, and these seem like switches on the run itself.